README.md

```diff
@@ -120,7 +120,7 @@ var corsOptions = {
 }
 
 app.get('/products/:id', cors(corsOptions), function (req, res, next) {
-  res.json({msg: 'This is CORS-enabled for a whitelisted domain.'})
+  res.json({msg: 'This is CORS-enabled for a allowlisted domain.'})
```

Comment:
```suggestion
  res.json({msg: 'This is CORS-enabled for an allowed domain.'})
```

Would this wording for for you? It would help move it that much further away from whitelist, I think.

Reply:
This sounds perfect :100:

WDYT @jkasun?